.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You should almost definitely exclude this file

Reply:
I'll put these in .gitignore for posterity :)